samples/bluetooth/iso_connected_benchmark/sample.yaml

```diff
@@ -0,0 +1,9 @@
+sample:
+  name: BLE ISO Connected
+  description: Bluetooth Low Energy ISO Connected Benchmark sample
+tests:
+  samples.bluetooth.iso_connected_benchmark:
+    build_only: true
```

Comment:
this is a benchmark so why build_only?

Reply:
The benchmark sample application doesn't run autonomously, so we can only build it.